src/tictacclojure/player.clj

```diff
@@ -1,28 +1,37 @@
 (ns tictacclojure.player
   (:require [tictacclojure.board :as board]
+            [tictacclojure.console-ui :as ui]
+            [tictacclojure.text-prompts :as prompts]
             [tictacclojure.minimax :as minimax]))
 
-(defn- console-in
-  []
-  (read-line))
+(defn- validate-text-move
```

Comment:
👍 I like the split between validating that it is _a number_ and its _a valid move on the board_.